### PR TITLE
Fixes agpl link

### PR DESCRIPTION
**Pull request by pachamaltese**

### CODE REVIEW

**Comment by pachamaltese** [comment]
@jennybc this PR has a small change to include the correct link to AGPL in the #* part

File: R/license.R [hunk 1]
```diff
@@ -72,6 +72,22 @@ use_agpl3_license <- function(name = find_name()) {
...
+  use_description_field("License", "AGPL-3 + file LICENSE", overwrite = TRUE)
+  use_license_template("AGPL-3", name)
+
+  use_template(
+    "year-copyright.txt",
+    save_as = "LICENSE",
+    data = license_data(name)
+  )
+}
+
```
Comment (jennybc) [inline R/license.R]: Why is this part of the diff? It was added in the previous PR, right?

Comment (pachamaltese) [reply R/license.R]: @jennybc the previous PR was merged, now I editerd directly onm github.com and I added an "a" to the url, therefore it fixes the link

**Comment by jennybc** [comment]
Thanks!

**Pull request merged**